tests/run.sh

```diff
@@ -114,6 +114,17 @@ function get_requirements() {
 
     chmod +x downloaded-tools/mender-cli
 
+    curl --fail "https://raw.githubusercontent.com/mendersoftware/mender/${MENDER_BRANCH}/support/modules-artifact-gen/directory-artifact-gen" \
+         -o downloaded-tools/directory-artifact-gen \
+         -z downloaded-tools/directory-artifact-gen
+
+    if [ $? -ne 0 ]; then
+        echo "failed to download directory-artifact-gen"
+        exit 1
+    fi
+
+    chmod +x downloaded-tools/directory-artifact-gen
```

Comment:
How come that this was missing before? I guess no one has run all tests locally for a while...